rick-morty-app-client/src/components/molecules/LanguageToggle/LanguageToggle.tsx

```diff
@@ -0,0 +1,33 @@
+import React from "react";
+import { useTranslation } from "react-i18next";
+
+const LanguageToggle = () => {
+  const { i18n } = useTranslation();
+  const { t }: { t: (key: string) => string } = useTranslation();
```

Comment:
use this hooks once and extract i18n and t, not seperatly

Reply:
poprawione

Reply:
jak poprawiasz, to klikaj "resolve conversation